pybossa/cache/users.py

```diff
@@ -357,6 +357,13 @@ def get_user_preferences(user_id):
     return get_user_pref_db_clause(user_pref, user_email)
 
 
+@memoize(timeout=ONE_DAY)
+def get_user_filters(user_id):
+    user_profile = get_user_profile_metadata(user_id)
+    user_profile = json.loads(user_profile) if user_profile else {}
```

Comment:
Is it intentional to use the same variable twice here?

Reply:
`user_profile = json.loads(user_profile) if user_profile else {}` parses the `user_profile` and assign the parsed result to `user_profile` again.